src/__tests__/__snapshots__/header.snapshot.tsx.snap

```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Is this auto generated code? Can we remove the comment?

Reply:
yeah this is auto generated